scalafix-tests/output/src/main/scala-3/test/organizeImports/DeduplicateImportees.scala

```diff
@@ -0,0 +1,7 @@
+package test.organizeImports
+
+import scala.collection.immutable.Vector
+import scala.collection.immutable.{Map => Dict}
+import scala.collection.immutable.{Set as _, *}
```

Comment:
The first line is [preserved](https://github.com/liancheng/scalafix-organize-imports/issues/127) while the second is generated by Scalafix, thus the inconsistency. We need to fix that - just not sure if it has to be in this PR?